src/styles/blocks/main-title.scss

```diff
@@ -0,0 +1,218 @@
+.main {
+
+  &__container {
+    padding: 0 20px;
+    margin-bottom: 60px;
+
+    @include onTablet {
+      padding: 0 35px;
+      margin-bottom: 100px;
+    }
+
+    @include onDesktop {
+      padding: 0 55px;
+      margin-bottom: 100px;
+    }
+
+    &--nevs {
```

Comment:
Consider using a more descriptive name for the modifier, like 'news' instead of 'nevs'.